include/ncengine/ecs/AnyComponent.h

```diff
@@ -74,12 +74,12 @@ class AnyComponent
         }
 
         /** @brief Invoke ComponentHandler::drawUI with the component instance, if it is set. */
-        void DrawUI()
+        void DrawUI(Entity self, ui::editor::EditorContext& ctx)
```

Comment:
Per our conversation - don't need 'self' passed in separately because 'selectedEntity' is already in `EditorContext`